Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/FileShareServiceTest.cs

```diff
@@ -125,5 +125,40 @@ public void WhenFSSDownloadFileAsyncThrowsException_ThenShouldExecuteCatch()
             Task<Stream> result = _fileShareService.FSSDownloadFileAsync("", "", "", CorrelationId);
             Assert.IsTrue(result.IsFaulted);
         }
+
+        [Test]
+        public void WhenFSSDownloadZipFileIsCalled_ThenShouldReturnByteArray()
```

Comment:
As per previous comment:
```suggestion
        public void WhenFSSDownloadZipAsyncFileIsCalled_ThenShouldReturnByteArray()
```

Reply:
Done